courses/machine_learning/deepdive2/structured/labs/5b_deploy_keras_ai_platform_babyweight.ipynb

```diff
@@ -0,0 +1,339 @@
+{
```

Comment:
<p>is this line too long. Not sure the conventions for bash.</p><p>Would "gsutil ls -d gs://${BUCKET}/babyweight/trained_model/2* | tail -1" suffice here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/675/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Fixed</p>